### PR TITLE
Fix jitpack publishing

### PR DESCRIPTION
**Pull request by vcoolish**
Fixes: #161

What changed:
- Fetch raw files before publish to jitpack

### CODE REVIEW

**Review by phuongtayga — COMMENTED** [review]
Abc

**Comment by phuongtayga** [comment]
> Abc

**Comment by phuongtayga** [comment]
Main

File: jitpack.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+before_install:
+  - rm -rf ./android/lib/src/main/res/raw/trust.js
```
Comment (hewigovens) [inline jitpack.yml:2]:
```suggestion
  - rm -rf android/lib/src/main/res/raw/trust.js
```

File: jitpack.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+  - mkdir -pv ./android/lib/src/main/res/raw/
```
Comment (hewigovens) [inline jitpack.yml:3]:
```suggestion
  - mkdir -pv android/lib/src/main/res/raw/
```

File: jitpack.yml [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+  - git lfs fetch --all
+  - git lfs checkout dist/trust-min.js
+  - mv ./dist/trust-min.js ./dist/trust_min.js
+  - cp ./dist/trust_min.js ./android/lib/src/main/res/raw/
```
Comment (hewigovens) [inline jitpack.yml:7]: "-" doesn't work in filename?
```suggestion
  - cp ./dist/trust-min.js android/lib/src/main/res/raw/trust_min.js

```

**Review by hewigovens — APPROVED** [review]

**Comment by hewigovens** [comment]
let's update readme too and disable publishing to GPR if jitpack works

**Pull request merged**